src/orderbook.ts

```diff
@@ -0,0 +1,34 @@
+export class Orderbook {
+  from: string;
+  to: string;
+  asks: Map<number, number>;
+  bids: Map<number, number>;
```

Comment:
Any reason why you are using the map type instead of a plain old JS object:
```
asks: { [key: number]: number }
bids: { [key: number]: number }
```

Reply:
Map has a bunch of convenience functions such as entries, values, get and is a little bit more concise to write that the good old dictionary type def.